account_invoice_supplier_self_invoice/views/report_self_invoice.xml

```diff
@@ -92,14 +92,18 @@
         priority="99"
     >
         <!-- Address switcher -->
-        <xpath expr="//t[@t-set='address']" position="replace">
+        <xpath
```

Comment:
It was improved on 14.0 in order to avoid the replacement, as it could give as problems with other modules, like sale

https://github.com/OCA/account-invoicing/pull/1418/files

It was not ported to 15. My error. Sorry, however, it might be necessary

Reply:
I don't think it was really needed, because in this new version the report is split up.
The `information_block` is only shown when the delivery address is different, but this is not possible on a vendor invoice.
But of course it is better to not replace the standard logic :-)

Reply:
@etobella Can you verify this one?